app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class CommentsController < ApplicationController
+  def index
+    @my_thread = MyThread.find(params[:my_thread_id])
+    @comment = Comment.where(my_thread_id: @my_thread.id).order(created_at: 'desc')
+    @myid = session[:user_id]
+  end
+
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @my_thread = MyThread.find(params[:my_thread_id])
+    @comment = Comment.new(comment_params)
+    @comment[:my_thread_id] = @my_thread.id
+    @comment[:user_id] = session[:user_id]
```

Comment:
```suggestion
    @comment.user_id = current_user.id
```

https://github.com/plataformatec/devise#controller-filters-and-helpers